docs/Files/UI/Solution-Manager/Library/Resources/Resources.md

```diff
@@ -0,0 +1,19 @@
+---
+lang: en-us
+title: Resources
+viewport: width=device-width, initial-scale=1.0
+---
+
+# Resources
+
+Available Resources in OpCon are shown in the following Grid under Library -> Resources.
+![Resource Grid](../../../../../Resources/Images/SM/Library/Resources/Resource-Grid.png "Resource Grid")
+
+Clicking the **Add** or **Edit** button bring up the following screen:
+![Resource Add](../../../../../Resources/Images/SM/Library/Resources/Resource-Add.png "Resource Add Screen")
+
+![White pencil/paper icon on gray circular background](<../../../../../Resources/Images/note-icon(48x48).png> "Note icon") **NOTE:** [The **Name** field is unique across Resources _and_ Thresholds
```

Comment:
Can we please use a notation like this for notes on a go-forward basis?

All notes in the format provided here will be phased out.

:::note
your text here
:::